extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -559,70 +668,91 @@ public function getMainPageIdForWikiId( $wikiId ) {
 
 	/**
 	 * Returns text from the main page of a provided wiki.
+	 *
 	 * @param int $wikiId
+	 *
 	 * @return string
 	 */
 	public function getMainPageTextForWikiId( $wikiId ) {
-		$params = array(
-				'controller' => 'ArticlesApiController',
-				'method' => 'getDetails',
-				'titles' => $this->getMainPageTitleForWikiId( $wikiId )->getDbKey()
-				);
-		$response = (new \ApiService)->foreignCall( $this->getDbNameForWikiId( $wikiId ), $params, \ApiService::WIKIA );
+		$params = [
+			'controller' => 'ArticlesApiController',
+			'method' => 'getDetails',
+			'titles' => $this->getMainPageTitleForWikiId( $wikiId )->getDbKey()
+		];
+		$response =
+			( new \ApiService )->foreignCall( $this->getDbNameForWikiId( $wikiId ), $params, \ApiService::WIKIA );
 		$item = \array_shift( $response['items'] );
+
 		return $item['abstract'];
 	}
 
 	/**
 	 * Returns text from the main page of a provided wiki.
+	 *
 	 * @param int $wikiId
+	 *
 	 * @return string
 	 */
 	public function getDescriptionTextForWikiId( $wikiId ) {
-		$response = (new \ApiService)->foreignCall(
+		$response = ( new \ApiService )->foreignCall(
 			$this->getDbNameForWikiId( $wikiId ),
-			array(
-					'action'      => 'query',
-					'meta'        => 'allmessages',
-					'ammessages'  => 'description',
-					'amlang'      => $this->getGlobalForWiki( 'wgLanguageCode', $wikiId )
-					)
-			);
-		return str_replace( '{{SITENAME}}', $this->getGlobalForWiki( 'wgSitename', $wikiId ), $response['query']['allmessages'][0]['*'] );
+			[
+				'action' => 'query',
+				'meta' => 'allmessages',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "allmessages" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)